dlk/python/dlk/templates/include/global.tpl.h

```diff
@@ -30,12 +30,32 @@ typedef int16_t   T_INT16;
 
 #define QUANTIZED_NOT_PACKED uint8_t
 
+enum class MemoryFormat {
+  HWC,
+};
+
+template <typename pack_type, MemoryFormat memory_format>
+class QuantizedPacked {
```

Comment:
Agreed with @antonionevado, `MemoryFormat` sounds more as an attribute of the container not packed type. Furthermore, at the current moment, you do not use it. So would suggest removing it at least for now. Then later if it happens that it is handy to tag packed type with we could easily introduce it again.

Reply:
Removed `MemoryFormat`.